app/views/compute_resources/show/_vmware.html.erb

```diff
@@ -6,3 +6,7 @@
   <td><%= _("Console passwords") %></td>
   <td><%= @compute_resource.set_console_password? ? _("Enabled") : _("Disabled") %></td>
 </tr>
+<tr>
+  <td><%= _("Caching") %></td>
+  <td><%= @compute_resource.caching_enabled ? _("Enabled") : _("Disabled") %></td>
```

Comment:
consider using an icon ? from http://www.patternfly.org/styles/icons/#_ maybe  pficon-ok and  pficon-warning-triangle-o?